### PR TITLE
Fix the broken E2E tests

### PR DESCRIPTION
**Pull request by ZhiminXiang**
The E2E tests are broken because the net-istio controller now only works with Istio 1.5 while the Istio addon is still 1.4.

### CODE REVIEW

**Comment by ZhiminXiang** [comment]
I logged into the cluster. All Knative services were not ready with the reason "IngressNotConfigured".

**Comment by ZhiminXiang** [comment]
Found below error logs 
```
{
 insertId: "fs4ncz4bm8zalhtko"  
 jsonPayload: {
  message: "k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:105: Failed to list *v1beta1.VirtualService: the server could not find the requested resource (get virtualservices.networking.istio.io)"   
  pid: "1"   
  source: "reflector.go:153"   
 }
 labels: {
  gke.googleapis.com/log_type: "system"   
  k8s-pod/app: "networking-istio"   
  k8s-pod/pod-template-hash: "5c9b57898c"   
  k8s-pod/serving_knative_dev/release: "v20200730-df95b90"   
 }
 logName: "projects/knative-boskos-25/logs/stderr"  
 receiveTimestamp: "2020-07-30T19:01:59.924423903Z"  
 resource: {
  labels: {
   cluster_name: "kdocs-e2e-cls1288909534055632900"    
   container_name: "networking-istio"    
   location: "us-central1"    
   namespace_name: "knative-serving"    
   pod_name: "networking-istio-5c9b57898c-9972g"    
   project_id: "knative-boskos-25"    
  }
  type: "k8s_container"   
 }
 severity: "ERROR"  
 sourceLocation: {
  file: "reflector.go"   
  line: "153"   
 }
 timestamp: "2020-07-30T19:01:58.491875Z"  
}
```

I suspect the installed Istio addon does not support v1beta1 yet.

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -16,13 +16,26 @@
...
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 
-function knative_setup() {
+function install_istio() {
+  ISTIO_STABLE=$(curl https://raw.githubusercontent.com/knative/serving/master/third_party/istio-stable)
+  echo ">> Bringing up Istio"
+  echo ">> Running Istio CRD installer"
+  kubectl apply -f "https://raw.githubusercontent.com/knative/serving/master/third_party/${ISTIO_STABLE}/istio-crds.yaml" || return 1
+  wait_until_batch_job_complete istio-system || return 1
+
```
Comment (mattmoor) [inline test/e2e-tests.sh:25]: Remove trailing whitespace:
```suggestion

```

File: test/e2e-tests.sh [hunk 2]
```diff
@@ -16,13 +16,26 @@
...
 
-function knative_setup() {
+function install_istio() {
+  ISTIO_STABLE=$(curl https://raw.githubusercontent.com/knative/serving/master/third_party/istio-stable)
+  echo ">> Bringing up Istio"
+  echo ">> Running Istio CRD installer"
+  kubectl apply -f "https://raw.githubusercontent.com/knative/serving/master/third_party/${ISTIO_STABLE}/istio-crds.yaml" || return 1
+  wait_until_batch_job_complete istio-system || return 1
+  
+  echo ">> Running Istio"
```
Comment (mattmoor) [inline test/e2e-tests.sh:26]: Remove trailing whitespace:
```suggestion
  echo ">> Running Istio"
```

**Pull request merged**